### PR TITLE
add UA "OperaCoast"

### PR DESCRIPTION
**Pull request by shokai**
Hi, I have added UA of Opera Coast.

> Mozilla/5.0 (iPhone; CPU iPhone OS 10_1_1 like Mac OS X) AppleWebKit/602.2.14 (KHTML, like Gecko) Coast/5.04.110603 Mobile/14B100 Safari/7534.48.3

It is Opera's iOS version.

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -74,6 +74,18 @@ describe('browser-info', function() {
     should(info().fullVersion).equal('12.16');
     done();
   });
+
+  it('should detect Coast', function(done) {
+    global.navigator = {
+      userAgent: 'Mozilla/5.0 (iPhone; CPU iPhone OS 10_1_1 like Mac OS X) AppleWebKit/602.2.14 (KHTML, like Gecko) Coast/5.04.110603 Mobile/14B100 Safari/7534.48.3'
+    };
+    should(info().os).equal('iOS');
+    should(info().name).equal('Coast');
```
Comment (shokai) [inline test/index.js]: should return `'OperaCoast'` ?

Comment (stevelacy) [inline test/index.js]: Yes, it should be OperaCoast

File: test/index.js [hunk 2]
```diff
@@ -74,6 +74,18 @@ describe('browser-info', function() {
     should(info().fullVersion).equal('12.16');
     done();
   });
+
+  it('should detect OperaCoast', function(done) {
+    global.navigator = {
+      userAgent: 'Mozilla/5.0 (iPhone; CPU iPhone OS 10_1_1 like Mac OS X) AppleWebKit/602.2.14 (KHTML, like Gecko) Coast/5.04.110603 Mobile/14B100 Safari/7534.48.3'
+    };
+    should(info().os).equal('iOS');
+    should(info().name).equal('OperaCoast');
```
Comment (shokai) [inline test/index.js]: now name is `OperaCoast`

**Comment by stevelacy** [comment]
thanks!

**Pull request merged**